### DISCUSSION TITLE
讓學生對社會議題表態，教育界到底在怕什麼？

### DISCUSSION THREAD

**Discussion by 110302550** [1101-GS4719-C/Python-Community-GS4719-C#1103] [General]
https://www.thenewslens.com/article/59261
我覺得知識的價值應該與社會相互連結，透過直接感受知識與現實之間的衝擊來體悟更深層次的價值，所以教育部應該在計畫中對於現代公民核心能力的養成，也有明確的目標，如培養學生關心社會、主動參與公共事務、面對新的挑戰與理解等。雖然教育界是一個保守的地方，而且大多數的教師還是主張學生不應該參與評論與立場選擇，但這也提出了教育中對於知識價值與社會相關的現實矛盾。

**Comment by Sylvie0601** [comment]
我覺得應該是要培養學生的獨立思考能力、判斷力，在現在這個外物紛擾是非多的時代，我覺得判斷事物、思考是很重要的，對於現在的人來說，不用經過思考就能得到結果的事情實在太容易了。
